doc/09-object-types.md

```diff
@@ -1425,6 +1425,11 @@ Configuration Attributes:
 
 ### IdoMySqlConnection <a id="objecttype-idomysqlconnection"></a>
 
+> **Note**
+>
+> This feature is DEPRECATED and may be removed in future releases.
+> Check the [roadmap](https://github.com/Icinga/icinga2/milestones).
```

Comment:
To have the changes properly show up in icinga.com, this PR has to be backported to support/2.13, where I wouldn't deprecate the IDO yet but rather only in the 2.14 release.

One would either have to remember this when backporting the PR, or probably better as it removes the opportunity to forget it during backporting: make a separate PR for this change that will only go in 2.14.

Reply:
I would immediately deprecate it as it is tied to the Icinga DB release and not an Icinga 2 version. The blog post will announce that as well.